src/runtime/composables/useFormGroup.ts

```diff
@@ -38,9 +46,12 @@ export const useFormGroup = (inputAttrs?: InputAttrs) => {
     }, 300)
 
     return {
+      inputId: formGroup?.inputId,
+      name: computed(() => inputProps?.name ?? formGroup?.name),
+      size: computed(() => inputProps?.size ?? formGroup?.size?.value ?? appConfig.ui.input.default.size),
+      color: computed(() => formGroup?.error?.value ? 'red' : inputProps?.color),
```

Comment:
It seems the color is no longer injected for Radio and Checkbox:
<img width="116" alt="CleanShot 2023-09-20 at 22 10 45@2x" src="https://github.com/nuxt/ui/assets/739984/0c0b08d7-7bad-4f1e-b012-a7d57db03f8c">
<img width="1122" alt="CleanShot 2023-09-20 at 22 10 52@2x" src="https://github.com/nuxt/ui/assets/739984/ce43db27-0843-4b81-9006-816506292212">

Reply:
Fixed it here: https://github.com/nuxt/ui/pull/704/commits/d4ac8ec5042bfb2d1977006409f7a5841ad3c978